### ISSUE TITLE
OCaml Operator Lookup

### ISSUE THREAD

**Issue by patricoferris** [ocaml/v2.ocaml.org#1223]
Recently @craigfe produced an excellent web application for [understanding OCaml's operators](https://www.craigfe.io/operator-lookup/). It was received [very well](https://twitter.com/_craigfe/status/1333365035779940353) [by many](https://discuss.ocaml.org/t/ann-operator-lookup-tool-for-ocaml/6882)! I think this would make a great addition in the form of a link in the learning section of OCaml.org and maybe in the future incorporate it directly in the site. I believe Craig is happy for the site to link to his work. 

I propose adding this to either the learning homepage (https://ocaml.org/learn/) or in one of the tutorials with a preference for the former. Where exactly it goes I'm unsure, perhaps the `Online Courses, Slides & Videos` could be transformed to an `External Learning` section (some of those videos/slides could perhaps be removed)? Happy for more thoughts and opinions on this. 

Thanks again for this Craig :))

**Comment by Aditi-1400** [comment]
I was just going through the list of issues, and came across this one, that, indeed is a really helpful application! :D  
 @patricoferris  I believe, a link to this can be added in, the external links section on this page : https://ocaml.org/learn/tutorials/?🤔

**Comment by patricoferris** [comment]
@Aditi-1400 I agree :)) I asked Craig some time ago and he was all up for linking it, feel free to submit a PR for this thanks 👍

**Comment by Aditi-1400** [comment]
Hey @patricoferris, I've made a pr which solves this issue. Also, do let me know if there's something else, I could contribute to. Always happy to help. :)
